pkg/routeagent_driver/handlers/kubeproxy/vxlan.go

```diff
@@ -301,3 +304,25 @@ func (kp *SyncHandler) createVxLANInterface(activeEndPoint string, ifaceType int
 
 	return nil
 }
+
+func (kp *SyncHandler) ensureLooseModeIsConfigured(iface string) error {
+	for i := 0; i < 10; i++ {
+		// Revisit: This is a temporary work-around to fix https://github.com/submariner-io/submariner/issues/2422
+		// Allow the vx-submariner interface to get initialized.
+		time.Sleep(100 * time.Millisecond)
+
+		rpFilterSetting, err := kp.netLink.GetReversePathFilter(iface)
+		if err == nil {
+			if bytes.Equal(rpFilterSetting, []byte("2")) {
+				return nil
+			}
+		}
```

Comment:
We should at least log the error here.

```suggestion
		if err == nil {
	            if bytes.Equal(rpFilterSetting, []byte("2")) {
		        return nil
		    }
		} else {
		    logger.Warningf("Error retrieving reverse path filter: %v", err)
		}
```

Reply:
TBH, it does not really help us in anyway even if we have the log message. But if you insist, I can add the log message.

Reply:
I wouldn't say I'm insisting - I think it's better to handle the error in some way rather than ignore it, especially since we're depending on successful retrieval to proceed.  Who knows - maybe something strange could cause retrieval to fail...